### ISSUE TITLE
JVMChaos: injection return but java process is failed 

### ISSUE THREAD

**Issue by GitHaoLi** [chaos-mesh/chaos-mesh#4362]
What version of Kubernetes are you using?
v1.24

What version of Chaos Mesh are you using?
2.6.1

What did you do? / Minimal Reproducible Example
JVMChaos is Inject , but java process is failed

![WX20240307-092951@2x](https://github.com/chaos-mesh/chaos-mesh/assets/18479621/47a549fe-0137-4a18-a499-3b5af36f7e0c)

What did you expect to see?
JVM return

java process logs 

![WX20240307-092905@2x](https://github.com/chaos-mesh/chaos-mesh/assets/18479621/3646501f-24f2-4197-901c-a713f8eb7c51)

chaos-daemon logs

![WX20240306-175729@2x](https://github.com/chaos-mesh/chaos-mesh/assets/18479621/9a125919-f869-459d-bca7-e97cef2949f4)

**Comment by STRRL** [comment]
![image](https://github.com/chaos-mesh/chaos-mesh/assets/20221408/61d69615-7ad2-46d0-8289-fc83647148b7)

I am not sure why Chaos Mesh use the port `0` here, it should be `9277` by default.

ref: https://chaos-mesh.org/docs/simulate-jvm-application-chaos/

The default value `9277` is set by mnutating webhook, but the webhook does work in your issue. please make sure the webhook part is working properly.


https://github.com/chaos-mesh/chaos-mesh/blob/d120d2db5ec569911b7521ec0a77a5403c338e2e/api/v1alpha1/jvmchaos_webhook.go#L26

**Comment by GitHaoLi** [comment]
@STRRL 
Yes I delete webhook , but I will try set port 9091 for jvm chaos 

controller error log is :
![WX20240320-111858@2x](https://github.com/chaos-mesh/chaos-mesh/assets/18479621/9ba895e3-8973-426f-8463-824193596724)
